lupa/tests/test.py

```diff
@@ -2697,9 +2703,6 @@ def test_number(self):
     def test_table(self):
         self._test_set_overflow_handler('{}')
 
-    def test_userdata(self):
-        self._test_set_overflow_handler('python.builtins.float')
-
```

Comment:
Why did this have to go?

Reply:
Because we now accept userdata as overflow handlers.

Reply:
We could test not POBJECT userdata in earlier versions Lua where newproxy is available, but not on the latest versions.